backend/app/routes/imports/__init__.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter, Header, HTTPException
+
+from app.config import settings
+
+router = APIRouter(prefix="/imports", tags=["Import Clearcuts"])
+
+
+def authenticate(x_imports_token: str = Header(default="")):
+    if x_imports_token != settings.IMPORTS_TOKEN:
+        raise HTTPException(status_code=401, detail="Invalid token")
+
+
+# Pull the child routes back into the parent
+from . import clearcut # noqa: F401 E402
```

Comment:
I want to get the child routes back into the exported router that is then mounted in main.py. I'm not sure if there is a better way to do this or not... My intuition tells me that a command that requires 2 ignores to qa _may_ not be the good way, but I don't see another path... Thoughts?